convert.go

```diff
@@ -137,6 +140,10 @@ func DialArgs(m ma.Multiaddr) (string, string, error) {
 				hostname = true
 				ip = c.Value()
 				return true
+			case ma.P_UNIX:
```

Comment:
function doc needs updating with this change.

Reply:
done!